doc/user_guide/edml_user_guide.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started With the Exasol Document Mapping Language (EDML)
+
+For creating a Virtual Schema for document data you have to define a mapping 
+from the document structure to a relational structure.
+This is done using the Exasol Document Mapping Language (EDML) 
+([reference](https://exasol.github.io/virtual-schema-common-ducument/schema_doc/index.html)).
+You have to defined the mapping in a JSON document, upload it to a bucket in BucketFS and reference 
+in the `CREATE VIRTUAL SCHEMA` call.
+
+This guide explains you how to define mappings these mappings in general.
+For data source specifics, check the corresponding virtual schema.
+Different data source use different data formats.
+In this guide we use JSON.
+  
+For mapping multiple document sets, you can create multiple files, 
+upload them to a folder and BucketFS and reference this folder. 
+
+The structure of the mapping follows the structure of the document data.
+
+## Example
+
+Given a DynamoDB table called `MY_BOOKS` that contains the following objects:
```

Comment:
In the examples, `MY_BOOKS` and `BOOKS` maybe difficult to distinguish. I think, you can keep them same `BOOKS` and update descriptions. Or add prefix for source and destination: `DYNAMODB_BOOKS` and `EXASOL_BOOKS`.